web/src/main/java/com/bazaarvoice/emodb/web/scanner/ScanUploader.java

```diff
@@ -96,33 +115,60 @@ private ScanPlan createPlan(String scanId, ScanOptions options) {
     }
 
     private void startScanUpload(String scanId, ScanStatus status) {
-        boolean scanCreated = false;
+        final AtomicReference<Boolean> scanCreated = new AtomicReference<>(false);
```

Comment:
`scanCreated` isn't used outside of the thread's runnable.  Why not just make it a local variable there?